blank-canvas-blocks/experimental-theme.json

```diff
@@ -171,6 +171,15 @@
 						"box-shadow": "none"
 					}
 				},
+				"paragraph": {
+					"dropcap": {
+						"typography": {
+							"font-family": "var(--wp--preset--font-family--base)",
```

Comment:
Nit pick but are we going with camel case or trying matching the CSS attribute?

I saw in [another PR a camel case](https://github.com/Automattic/themes/pull/3437/files#diff-6b32b96741b2b4b9e621f53f3709cb5e5dcbb41c0467f83a2f30c99577f724b8R149), and it made me think we should be consistent with what Gutenberg is doing.

Reply:
+1 on this, I opened a PR to change to camel case here https://github.com/Automattic/themes/pull/3444